subprojects/testing-base/src/main/java/org/gradle/api/tasks/testing/TestDescriptor.java

```diff
@@ -43,6 +43,12 @@
     @Incubating
     String getDisplayName();
 
+    /**
+     * Whether or not the class has a @DisplayName annotation.
+     * @return true if the class has @DisplayName annotation; false otherwise.
+     */
+    boolean hasDisplayNameAnnotation();
```

Comment:
:x: This shouldn't be part of the public API: the `TestDescriptor` is ecosystem agnostic and it doesn't make sense to talk about a display name annotation here. If we need this, it should be on an internal interface.

Reply:
I've remove the method from the TestDescriptor. I found it to be an unnecessary definition.